src/Schema.js

```diff
@@ -802,17 +804,23 @@ const makeDescriptor = function (prop, schema, opts) {
     // TODO: Make it so tracking can be turned on for all properties instead of
     // only per-property
     if (track && !_get(creatingPath)) {
+      /* previous is versioned on database commit
+       * props are versioned on set()
+       * TODO? Option to disable props versioning (commit only versioning)
```

Comment:
I meant whether it's worth having an option that only updates changeHistory on commit() / save() e.g. as an array of previous ?